README.md

```diff
@@ -27,33 +26,21 @@
 
 # What is Nx?
 
-🔎 **Nx is a set of Angular CLI power-ups for modern development.**
+🔎 **Extensible Dev Tools for Monorepos.**
 
 ## Nx Helps You
 
 ### Use Modern Tools
 
-Using Nx, you can add Cypress, Jest, Prettier, and Nest into your dev workflow. Nx sets up these tools and allows you to use them seamlessly. Nx fully integrates with the other modern tools you already use and love.
+Using Nx, you can add TypeScript, Cypress, Jest, Prettier, and Nest into your dev workflow. Nx sets up these tools and allows you to use them seamlessly. Nx fully integrates with the other modern tools you already use and love.
 
 ### Build Full-Stack Applications
 
-With Nx, you can build full-stack applications using Angular and Node.js frameworks such as Nest and Express. You can share code between the frontend and the backend. And you can use the familiar `ng build/test/serve` commands to power whole dev experience.
+With Nx, you can build full-stack applications using modern frameworks. You can share code between the frontend and the backend. And you can use the same `build/test/serve` commands to power whole dev experience.
 
-### Develop Like Google
+### Build like Google, Facebook, and Microsoft
 
-With Nx, you can develop multiple full-stack applications holistically and share code between them all in the same workspace. Nx provides advanced tools which help you scale your enterprise development. Nx helps enforce your organization’s standards and community best practices.
-
-## A la carte
-
-Most importantly, you can use these power-ups a la carte. Just want to build a single Angular application using Cypress? Nx is still an excellent choice for that.
-
-## Does it replace Angular CLI?
-
-Nx **is not** a replacement for Angular CLI. **An Nx workspace is an Angular CLI workspace.**
-
-- You run same `ng build`, `ng serve` commands.
-- You configure your projects in `angular.json`.
-- Anything you can do in a standard Angular CLI project, you can also do in an Nx workspace.
+With Nx, you can develop multiple full-stack applications holistically and share code between them all in the same workspace. Nx provides advanced tools which help you scale your enterprise development. Nx also helps enforce your organization’s standards and community best practices.
 
 # Getting Started
```

Comment:
We should recommend installing `@nrwl/cli` globally if we are going to use the `nx` cli in our documentation.

Reply:
`create-nx-workspace` will show a message recommending you to install it globally. Do you think it is enough?

Reply:
It's worth mentioning in the README + our guides as well.